test/e2e/index.ts

```diff
@@ -34,15 +43,68 @@ test('admin smoke test', withConductor(ADMIN_PORT, async t => {
 
   await admin.attachAppInterface({ port: 0 })
   await admin.deactivateApp({ installed_app_id })
-  const dnas = await admin.listDnas()
-  t.equal(dnas.length, 0)
+
+  let dnas = await admin.listDnas()
+  t.equal(dnas.length, 1)
 
   const activeApps3 = await admin.listActiveApps()
   t.equal(activeApps3.length, 0)
   // NB: missing dumpState because it requires a valid cell_id
+
+  // install from hash
+  const newHash = await admin.registerDna({
```

Comment:
Quick Question: 
Just to confirm.
If I use this `newHash` and run another `admin.registerDna({source: {newHash}, properties: {uuid: 111})`
Will this new DNA that gets installed have the old properties of the newHash DNA? or is the source code the same as the first hash installed?

Reply:
It should chain.  I.e. it should have the properties of the parent, not the root.